modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -50,7 +50,9 @@ var FilterRule = React.createClass({
 				"lessThanEqual" : "<=",
 		    	"greaterThanEqual" : ">=",
 		    	"startsWith" : "startsWith",
-		    	"contains" : "contains"
+		    	"contains" : "contains",
+		    	"isNull" : "isNull",
+		    	"isNotNull" : "isNotNull"
```

Comment:
From my experience, other software use the words "empty" and "not empty" for mortal users.